Stop keeping messages in unacked state

renq opened the pull request:
Here is explanation why:
https://github.com/Codeception/module-amqp/issues/11

Commit:
Stop keeping messages in unacked state

Naktibalda commented:
`grabMessageFromQueue` has the same issue, right?

Your change broke `countQueue` test, please update it.
It definitely requires major version bump.

Commit:
For backward compatibility, method seeMessageInQueueContainsText consumes the message instead of returning it

renq commented:
I fixed the tests.

I also changed `nack($requeue=true)` to `ack()`. I found in the comment above method `seeMessageInQueueContainsText` a sentence "This method drops a message from queue". If this was the intention of the author of this method, `ack()` seems to fit here better and which is more important, in my opinion, it doesn't break backward compatibility. I would treat my change as a bugfix.

Regarding grabMessageFromQueue - I think there is no bug here. It's a feature. :) This method returns `\PhpAmqpLib\Message\AMQPMessage`, so users can do whatever they want with that. They can leave it unacknowledged by doing nothing or ack/nack the message.
